Fibonacci.java

```diff
@@ -0,0 +1,18 @@
+public class Fibonacci {
+
+	public static void main(String[] args) { 
+		int nthFib = Integer.parseInt(args[0]);
+		System.out.println(new Fibonacci().findNthFib(nthFib));
+	}
+
+	private int findNthFib(int nthFib) {
```

Comment:
NIt: This parameter should be named something other then the name of the function, because its recursive